app/Exceptions/InvalidSubscriptionTypeException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+use RuntimeException;
+use Throwable;
+
+final class InvalidSubscriptionTypeException extends RuntimeException
+{
+    /**
+     * InvalidSubscriptionTypeException constructor.
+     *
+     * @param int $code
+     * @param string $message
+     * @param \Throwable|null $previous
+     */
+    public function __construct(String $message, Int $code = 0, ?Throwable $previous = null)
```

Comment:
Kindly lower case the type hint declaration for primitive data types instead of `String` use `string` and instead of `Int` use `int` instead thanks.